pkg/logcli/client/client.go

```diff
@@ -209,27 +217,40 @@ func (c *DefaultClient) doRequest(path, query string, quiet bool, out interface{
 	}
 
 	var resp *http.Response
-	attempts := c.Retries + 1
+
 	success := false
 
-	for attempts > 0 {
-		attempts--
+	bkcfg := backoff.Config{
+		MinBackoff: time.Duration(c.BackoffConfig.MinBackoff) * time.Second,
+		MaxBackoff: time.Duration(c.BackoffConfig.MaxBackoff) * time.Second,
+		// 0 max-retries for backoff means infinite number of retries.
+		MaxRetries: c.Retries + 1,
```

Comment:
So, if `retries` is 0. The max retries is 1. Does this mean we run only once because `backoff.Ongoing()` will be true the first time but false the second time?